cluster/kubectl.sh

```diff
@@ -18,6 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+echo "-=-=-=-=-=-=-=-=-=-="
+echo "NOTE:"
+echo "kubectl.sh is deprecated and will be removed soon."
+echo "please replace all usage with calls to the kubectl"
+echo "binary and ensure that it is in your PATH." 
```

Comment:
I don't see the ```--context``` bits for GKE, for ubuntu, the ```--server``` flag is mostly deprecated in favor of kubectl contexts, so I think that it's ok to just drop these.  People should have the appropriate context files in ```.kube/config```